keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -386,18 +384,35 @@ logs or events for this purpose.
   The usage of this feature requires the manual step of applying a taint
   so the operator should be the one applying it.
 
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [X] API .status
+  If it works, pods in the stateful workload should be re-scheduled to another
+  running node. `Phase` in Pod `Status` should be `Running` for a new Pod
+  on the other running node.
+  If not, check the pod status to see why it does not come up.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
   - [X] Metrics
     - Metric name: 
-      - We can add new metrics `deleting_pods_total`, `deleting_pods_error_total`
+      - We added new metrics `deleting_pods_total`, `deleting_pods_error_total`
       in Pod GC Controller.
       For Attach Detach Controller, there's already a metric:
-      attachdetach_controller_forced_detaches
-      It is also useful to know how many nodes have taints. We can explore with [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) which generates metrics about the state of the objects.
+      `attachdetach_controller_forced_detaches`
```

Comment:
Would it be helpful in the metric to distinguish the reason why we forced detached? ie node shutdown vs node unhealthy?

Reply:
@msau42 This was discussed [here](https://github.com/kubernetes/enhancements/pull/3320#discussion_r974785946) and [here](https://github.com/kubernetes/enhancements/pull/3320#discussion_r975687210). We discussed and decided a new metric is not needed, instead we could explore [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics). I added metrics from kube-state-metrics.

Reply:
I think adding a reason to force detach metric would be useful. With the node taint metric, we can only see that a node is tainted, but not that any volumes were detached because of it.

Reply:
Added.

Reply:
Did you push the update?

Reply:
Yes.  Lines 412-413. https://github.com/kubernetes/enhancements/pull/4029/files#diff-0225593bb1191b37cc24ad60c172668c3df10b62f2fd748ceb1bbe85ddf078ceR412-R413